src/server.js

```diff
@@ -1,11 +1,13 @@
 // @flow
+/* eslint-disable global-require,import/no-dynamic-require */
 
+import { log } from 'util';
```

Comment:
NB: this method is [deprecated](https://nodejs.org/docs/latest-v8.x/api/util.html#util_util_log_string). I have used [`bunyan`](https://github.com/trentm/node-bunyan) which did what we needed, but perhaps there are better alternatives these days.